lib/erubi.rb

```diff
@@ -143,7 +147,7 @@ def initialize(input, properties={})
             add_text(rspace)
           end
         when '%'
-          add_text("#{lspace}#{prefix||='<%'}#{code}#{tailch}#{postfix||='%>'}#{rspace}")
+          add_text("#{lspace}#{prefix||=literal_prefix}#{code}#{tailch}#{postfix||=literal_postfix}#{rspace}")
```

Comment:
I don't think you need the `prefix||=` and `postfix||=` anymore.

Reply:
Cool call, I didn't even notice that. Removed.